xdocs/usermanual/correlation.xml

```diff
@@ -0,0 +1,290 @@
+<?xml version="1.0"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to you under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!DOCTYPE document
+[
+<!ENTITY sect-num '25'>
+<!ENTITY hellip   "&#x02026;" >
+]>
+<document id="$Id$" prev="curl.html" next="history_future.html" >
+
+<properties>
+  <title>User's Manual: Auto-Correlation</title>
+</properties>
+
+<body>
+
+<section name="&sect-num;. Auto-Correlation" anchor="auto_correlation">
+<p>Auto-Correlation feature detects the dynamic parameters and adds response data extractors for them.<br/>
+A Test Plan using HTTP(S) requests can be correlated using this functionality.
+</p>
+
+<subsection name="&sect-num;.1 How to perform Auto-Correlation" anchor="how_to_correlation">
+<p>
+There are two ways to perform Auto-Correlation.<br/>
+<ul>
+<li><a href="#jmx_file">JMX File</a></li>
+<li><a href="#import_rule">Import Rule</a></li>
+</ul>
+<p id="important_note"><note> Test Plan must be recorded using the documentation shared
+<a href="jmeter_proxy_step_by_step.html">here</a>.
+</note>
+</p>
+</p>
+<subsection name="&sect-num;.1.1 JMX File" anchor="jmx_file">
+<p>
+Using a JMX Test Plan.
+</p>
+<p><b>Steps</b></p>
+
+1.Record a Test Plan for a testing scenario. Save it (say TestPlan_recording_one.jmx).<br/>
+2.Record another Test Plan with the same scenario.
```

Comment:
Where is this second test plan used ? I don't see it referenced later, am I missing something ?